src/pages/components/menu/usage.mdx

```diff
@@ -0,0 +1,572 @@
+
+| Modal size             | Height (px / rem)                                                                                                                                                                                                                                         |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Enabled                | When an action in the list is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component.                                                                                          |
+| Hover                  | When a user’s mouse cursor is hovering over the menu item in menus or submenus                                                                                                                                                                            |
+| Focus                  | When a user tabs to or clicks on a menu trigger. This will open up the menu list. The first item in the menu list has the focus state as default.                                                                                                         |
+| Focus and hover        | When an item is in focus state but the user’s mouse cursor is hovering over the item field                                                                                                                                                                |
+| Danger hover           | When an action could have destructive effects on the user’s data (for example, delete or remove)                                                                                                                                                          |
+| Danger hover and focus | When a user tabs to a destructive menu item that has the danger hover state                                                                                                                                                                               |
+| Disabled               | When an action is not able to be performed at a certain time, but will be possible to be performed by that user, at another point in time. Actions that are permissions-based should be hidden if it is never possible for a user to perform that action. |
+
+### Interactions
+
+#### Mouse
+
+Once the menu opens by clicking on a menu button trigger or right-clicking on an
+area, users can:
+
+- Select or deselect an item by clicking on it
+- Hover over the item with the caret icon to reveal additional options or
+  actions from that item
+- Click outside of the menu area to close it
+
+<Row>
+<Column colLg={8}>
+
+![Mouse interactions on menus](images/usage-interactions-mouse.png)
```

Comment:
This image has a spacing issue below the "Share with" item.
It also needs to have the caret icon at `$icon-primary` and the word "Product team" should be `$text-primary`.

Reply:
Also "Databases" and "Resources" are highlighted in the treeview, is this a mistake?

Reply:
Still looks like there is an issue with this image^